packages/@uppy/utils/src/NetworkError.js

```diff
@@ -0,0 +1,13 @@
+class NetworkError extends Error {
+  constructor (error = null, xhr = null) {
+    super(error.message)
```

Comment:
is there a reason we are not setting  `"This looks like a network error ..."` as the primary message instead? And append the source error? Something like this:

```js
super(`This looks like a network error, the endpoint might be blocked by an ISP or a firewall. Source error [${error}]`)
```

That way, we won't have to worry about the punctuation and the concatenation done below `error.message + '. This looks ...'` which can be error prone for some error messages that already contain a period '.', or for cases where `error` is `null`.

Reply:
Totally agree, changed! Thank you.